assets/i18n/ar.json

```diff
@@ -359,7 +359,6 @@
 	"I18N_PLAYER_IS_PRIVATE": "الاستكشاف خاصة.",
 	"I18N_PLAYER_LAST_UPDATED_TOOLTIP": "آخر تحديث",
 	"I18N_PLAYER_LEARN_AGAIN_BUTTON": "تعلم مرة أخرى",
-	"I18N_PLAYER_LEAVE_FEEDBACK": "ترك تغذية مرتدة للمؤلفين...",
```

Comment:
What is the plan for these translations? I imagine we will need to add them back at some point, right?

Reply:
@U8NWXD These will be updated by translatewiki contributors.